WorkingWithAzureDB/WorkingWithAzureDB/CustomerAddress.cs

```diff
@@ -11,15 +11,16 @@ namespace WorkingWithAzureDB
 {
     using System;
     using System.Collections.Generic;
-    
+    // Class to define the address of a Customer linked to address
```

Comment:
you can use class comments for this. see [here](https://docs.microsoft.com/en-us/dotnet/csharp/codedoc)